server/server.go

```diff
@@ -139,7 +142,9 @@ func New(cfg Config) (*Server, error) {
 		RegisterInstrumentation(router)
 	}
 	httpMiddleware := []middleware.Interface{
-		middleware.Tracer{},
+		middleware.Tracer{
+			Options: cfg.OpentracingOptions,
```

Comment:
Maybe the caller should pass in a middleware.Interface to act as Tracer, similar to how Log is done?

Reply:
What do you mean? Could you explain? The `Log` is being passed a `logger`...